documentation/modules/proc-accessing-kafka-using-routes.adoc

```diff
@@ -5,16 +5,23 @@
 [id='proc-accessing-kafka-using-routes-{context}']
 = Accessing Kafka using OpenShift routes
 
+This procedure describes how to access a Strimzi Kafka cluster from an external client outside of OpenShift using routes.
+
+To connect to a broker, you need a hostname (advertised address) for the route _bootstrap address_,
+as well as the certificate used for authentication.
+
+Routes are used for HTTP access, so the port used to connect is always 443.
```

Comment:
Again, we should either remove the HTTP reference or explain the details about TLS passthrough masking the Kafka traffic as HTTPS.

Reply:
> For access using routes, the port is always 443.